pg_query.pb.go

```diff
@@ -17988,7 +17988,7 @@ type RTEPermissionInfo struct {
 
 	Relid         uint32   `protobuf:"varint,1,opt,name=relid,proto3" json:"relid,omitempty"`
 	Inh           bool     `protobuf:"varint,2,opt,name=inh,proto3" json:"inh,omitempty"`
-	RequiredPerms int64    `protobuf:"varint,3,opt,name=required_perms,json=requiredPerms,proto3" json:"required_perms,omitempty"`
+	RequiredPerms uint64   `protobuf:"varint,3,opt,name=required_perms,json=requiredPerms,proto3" json:"required_perms,omitempty"`
```

Comment:
See https://github.com/pganalyze/libpg_query/commit/6f269718fb7095ac88a595d7d36390511d27297b for context